### PR TITLE
Fix all problems on PHPStan Level 0

### PR DESCRIPTION
**Pull request by szepeviktor**
From #14
Remove redundant PHPDoc tags from a handful of files.

### CODE REVIEW

**Comment by leoloso** [comment]
Thanks for this PR. I'll check it out tomorrow

File: phpstan.neon.dist [hunk 1]
```diff
@@ -1,4 +1,13 @@
+includes:
+    - vendor/szepeviktor/phpstan-wordpress/extension.neon
 parameters:
-	level: 5
-	paths:
-		- src
+#    level: 5
+    level: 0
```
Comment (leoloso) [inline phpstan.neon.dist:5]: Hi Viktor, I want to have level 5 executed, but the configuration has been changed to 0. Is your idea to start with 0, fix all the errors, then switch to 1, fix all the errors, and so on until reaching 5?

File: phpstan.neon.dist [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+    scanFiles:
+        # fullUrl() comes from this file
+        - vendor/getpop/migrate-engine/migrate/library/functions/utils.php
+    paths:
+        - src/
+    ignoreErrors:
+        # TODO Tidy up abstract methods.
+        - '#^Call to an undefined method GraphQLAPI\\GraphQLAPI\\Blocks\\AbstractBlock::getScriptDependencies\(\)\.$#'
```
Comment (leoloso) [inline phpstan.neon.dist:13]: I have fixed this error in `master`, removing `getScriptDependencies`. Please sync your branch and re-execute phpstan

File: phpstan.neon.dist (hunk 1 above)
Comment (szepeviktor) [reply phpstan.neon.dist:5]: Yes!

**Comment by leoloso** [comment]
I just uploaded the new `composer.lock` to `master`, please sync your branch and run phpstan again. You should not get the errors with `fullUrl` and `getScriptDependencies` anymore

**Comment by leoloso** [comment]
As a general rule, if phpstan gives any error concerning code under any `migrate-...` package, I must first migrate that code to the corresponding `PSR-4` package. I should do it myself, since I know the code in detail. Just let me know whenever it happens, so I can fix it.

File: phpstan.neon.dist (hunk 2 above)
Comment (szepeviktor) [reply phpstan.neon.dist]: Done.

**Comment by szepeviktor** [comment]
> You should not get the errors with fullUrl and getScriptDependencies anymore

https://github.com/GraphQLAPI/graphql-api/blob/master/src/Clients/CustomEndpointClientTrait.php#L24

Houston! We have a problem.

**Comment by leoloso** [comment]
Sorry! My mistake!

Fixed now, please sync again

**Comment by szepeviktor** [comment]
Done.

**Comment by leoloso** [comment]
Magnificent! Merging now :)

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**